frontend/src/metabase/query_builder/components/expressions/ExpressionWidget.tsx

```diff
@@ -233,15 +233,19 @@ export const ExpressionWidget = <Clause extends object = Lib.ExpressionClause>(
       {withName && (
         <FieldWrapper>
           <FieldLabel htmlFor="expression-name">{t`Name`}</FieldLabel>
-          <Input
+          <TextInput
+            classNames={{
```

Comment:
using `fw=bold` doesn't work since the input seems to have its own styles for `font-weight` set, so we have to hit the specific selector here